project_timesheet_holidays_contract/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2023 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Project Timesheet Holidays Contract",
+    "summary": "Schedule when on time Off with employee contracts taken into account",
```

Comment:
`summary` is a bit unclear, IMHO (but so is [project_timesheet_holidays](https://github.com/odoo/odoo/blob/17.0/addons/project_timesheet_holidays/__manifest__.py#L8)'s summary).
What do you think of something of the like of 
```
"summary": "Automatically log employees' leave in timesheets according to their contracts.",
```